.gitignore

```diff
@@ -13,14 +12,16 @@ docs/
 lib/
 es/
 test-env/
-.idea
```

Comment:
Any reason this was removed?

Reply:
I think it was a mistake, sorry about that fixing it now

Reply:
no it was not a mistake because ".idea" exists on the top, am i correct?

Reply:
You're right, I didn't see that :)